_docs/7_SubsquidArchive/docker-compose.yml

```diff
@@ -78,3 +78,16 @@ services:
       interval: 30s
       retries: 5
 
+  datadog:
+    image: datadog/agent
+    environment:
+      # replace DATADOG_API_KEY with real API key (can be genarated at https://app.datadoghq.com/organization-settings/api-keys)
+      - DD_API_KEY=DATADOG_API_KEY
+      - DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true
+      - DD_LOGS_ENABLED=true
+      - DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL=true
+      - DD_CONTAINER_EXCLUDE=name:datadog-agent
```

Comment:
This is more often done as a map, rather than array, which makes it more natural, see examples above in this very file